test/04-translations.test.ts

```diff
@@ -268,17 +270,18 @@ describe('04 native translation tests', function () {
 			});
 			assertExists(v2StudentCreated);
 
-			// translate the objects and test for deep equality
-			vLatestStudent.studies_at__campus =
-				vLatestStudent.studies_at__faculty[0].name;
-			delete vLatestStudent.studies_at__faculty;
-
 			expect(v2StudentCreated.computed_field).to.equal('v2_computed_field');
 			expect(vLatestStudent.computed_field).to.equal('latest_computed_field');
 			delete v2StudentCreated.computed_field;
 			delete vLatestStudent.computed_field;
 
-			expect(v2StudentCreated).to.deep.equal(vLatestStudent);
+			// translate the objects and test for deep equality
+			const { studies_at__faculty, ...vTranslatedStudent } = {
+				...vLatestStudent,
+				studies_at__campus: vLatestStudent.studies_at__faculty[0].name,
+			};
+
+			expect(v2StudentCreated).to.deep.equal(vTranslatedStudent);
```

Comment:
Would feel slightly easier to digest for me if it was split into two phases, but meh
```suggestion
			const { studies_at__faculty, ...vTranslatedStudent } = vLatestStudent;

			expect(v2StudentCreated).to.deep.equal({
				...vTranslatedStudent,
				studies_at__campus: vLatestStudent.studies_at__faculty[0].name,
			});
```